src/main/java/com/amihaiemil/zold/RtWallet.java

```diff
@@ -110,11 +113,33 @@ public double balance() throws IOException{
     public void pay(
         final String keygap, final String user,
         final double amount, final String details
-    ) {
-        throw new UnsupportedOperationException(
-            "Not yet implemented. If you can contribute please, do it at "
-            + "https://github.com/amihaiemil/zold-java-client"
+    ) throws IOException {
+        
+        final HttpPost pay = new HttpPost(
```

Comment:
@ammaratef45 it's not the best name for variable, because it's the same as this method name and doesn't describe the object.

Reply:
@g4s8 what about payRequest?

Reply:
@ammaratef45 `payRequest` or maybe just `request` would be good